addon/styles/_colors.scss

```
@@ -1,5 +1,175 @@
-// See https://zeroheight.com/8dd127da7/p/839645-couleurs/b/21317a
+// See https://www.figma.com/file/8RJ3aCSfdeQ8AZZVBBYKS8/Design-System-Pix?node-id=16%3A2
 
+//// SOLID
+// Primary
+$pix-primary: #3D68FF;
+$pix-primary-80: #000E87;
+$pix-primary-70: #0D25B3;
+$pix-primary-60: #2044DC;
+$pix-primary-40: #5B94FF;
+$pix-primary-30: #88BEFF;
+$pix-primary-20: #C4E6FF;
+$pix-primary-10: #DCF1FF;
+$pix-primary-5: #F2F9FF;
```

Je trouve plus lisible de garder l'ordre qu'on trouve dans Figma

```suggestion
$pix-primary-5: #F2F9FF;
$pix-primary-10: #DCF1FF;
$pix-primary-20: #C4E6FF;
$pix-primary-30: #88BEFF;
$pix-primary-40: #5B94FF;
$pix-primary: #3D68FF;
$pix-primary-60: #2044DC;
$pix-primary-70: #0D25B3;
$pix-primary-80: #000E87;
```